.eslintrc

```diff
@@ -22,7 +22,7 @@
         "eqeqeq": 2,
         "eol-last": 2,
         "indent": [2, 4],
-        "linebreak-style": [2, "unix"],
+        "linebreak-style": [1, "unix"],
```

Comment:
When git pulls from GitHub to Windows, it converts the linebreaks to Windows-style.  When you push back from Windows it converts back.  So (at least in theory) Windows and Linux developers each see their own line endings, but the repository only sees Linux line endings.  So having Linux and the CI builds check line endings is fine, but it breaks the tests when run by a Windows developer.  There are discussions at eslint about having environment-dependent rules, but they haven't come to anything yet.  So if we make it a warning, the CI and Linux developers will see there is a problem (if it gets past the auto-convert from Git) in that they will get any warnings at all.  Windows developers, second rate citizens that we are :), will just have to ignore the multiple-thousand warnings, but at least it won't make pre-commit tests fail.